src/link/index.ts

```diff
@@ -186,8 +186,8 @@ async function shamefullyFlattenTree(
     outdatedPkgs: {[pkgId: string]: string},
   },
 ): Promise<{[alias: string]: string[]}> {
-  const pkgIdByAlias = {}
-  const aliasByPkgId: {[pkgId: string]: string[]} = {}
+  const pkgPathByAlias = {}
```

Comment:
it is called [dependency path](https://github.com/pnpm/spec/blob/master/dependency-path.md) throughout the codebase, so better keep that convention